services/skus/service_test.go

```diff
@@ -8,14 +8,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/brave-intl/bat-go/libs/datastore"
```

Comment:
issue: Something is up with the settings of your IDE. This is not the sorting we want, please revert the change.